### PR TITLE
PLATO-206: Clean up help link/icon css rules and enhance for better accessibility

### PR DESCRIPTION
**Pull request by eslawski**
Resolves PLATO-206

There are currently two types of "help" links that can appear in modals across Artstor. 

1) Text based links:
image

2) Icon based links
image

Some of the styles were duplicated and others were colliding with eachother. To help clean things up I moved the two different styles to the top level scss file, and created new classes `help-link-text` and `help-link-icon` for referencing them.

This PR also ensures we are meeting the accessibility requirements outlined in PLATO-206

### CODE REVIEW

File: src/app/modals/add-to-group/add-to-group.component.scss [hunk 1]
```diff
@@ -59,27 +59,6 @@
         .wrapper {
             display: inline-flex;
 
-            .help-link {
```
Comment (eslawski) [inline src/app/modals/add-to-group/add-to-group.component.scss]: This block of css also appears in _modals.scss. Removed it here and moved it to the top level app.scss

File: src/app/modals/share-link-modal/share-link-modal.component.pug [hunk 2]
```diff
@@ -27,5 +27,5 @@
               | {{ 'SHARE_IMG_LINK_MODAL.COPY_TO_CLIPBOARD' | translate }}
             .copy-status-msg {{ copyHTMLStatusMsg }}
       .modal-footer
-        button.btn.btn-link.with-pad((click)="showHelp()", tabindex="0") {{ 'SHARE_IMG_LINK_MODAL.HELP' | translate }}
```
Comment (eslawski) [inline src/app/modals/share-link-modal/share-link-modal.component.pug]: Didn't really see the need for a function to handle linking to another page. I converted it to a link, let me know if I could be missing something though

File: src/app/modals/share-link-modal/share-link-modal.component.ts [hunk 3]
```diff
@@ -123,9 +123,4 @@ export class ShareLinkModal implements OnInit, AfterViewInit {
     document.body.removeChild(textArea);
   }
 
-  // Doesn't get called on SSR application
-  public showHelp(): void{
```
Comment (eslawski) [inline src/app/modals/share-link-modal/share-link-modal.component.ts]: unused

File: src/app/uploader/uploader.component.scss [hunk 4]
```diff
@@ -65,14 +65,9 @@
       color: #444;
       letter-spacing: 0.4px;
   }
-  .help{
```
Comment (eslawski) [inline src/app/uploader/uploader.component.scss]: renamed to `.help-link-text` and moved to `app.scss`

File: src/app/modals/add-to-group/add-to-group.component.pug [hunk 5]
```diff
@@ -65,7 +65,7 @@
             .modal-footer
               ng-container(*ngIf="!serviceResponse.success")
                 .wrapper
-                  a.help-link.mr-2(href="http://support.artstor.org/?article=create-and-manage-image-groups", target="_blank", tabindex="13", aria-label="Learn more about groups", (keydown.tab)="$event.stopPropagation(); $event.preventDefault(); startModalFocus()") ?
+                  a.help-link-icon.mr-2(href="http://support.artstor.org/?article=create-and-manage-image-groups", target="_blank", tabindex="13", aria-label="Learn more about groups", (keydown.tab)="$event.stopPropagation(); $event.preventDefault(); startModalFocus()") ?
```
Comment (NazimAli2017) [inline src/app/modals/add-to-group/add-to-group.component.pug]:
```suggestion
                  a.help-link-icon.mr-2(href="https://support.artstor.org/?article=create-and-manage-image-groups", target="_blank", tabindex="13", aria-label="Learn more about groups", (keydown.tab)="$event.stopPropagation(); $event.preventDefault(); startModalFocus()") ?
```

**Pull request merged**